packages/sn-filter-pane/package.json

```diff
@@ -50,10 +50,10 @@
     "@babel/preset-typescript": "^7.18.6",
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
-    "@nebula.js/cli": "4.6.1",
-    "@nebula.js/cli-build": "4.6.1",
-    "@nebula.js/cli-sense": "4.6.1",
-    "@nebula.js/cli-serve": "4.6.1",
+    "@nebula.js/cli": "4.7.0",
```

Comment:
I saw there is a version 4.8.0 of nebula out there already, might as well update to that version maybe? But then we need yet another nebula version to get your "fill height" in as well, right :)

Reply:
Yes, good point. I will actually bump it to `4.8.1` (just released) which contains the changes required for this feature so I won't have to bump it again. Will push shortly.